src/btree/bt_sync.c

```diff
@@ -68,15 +68,18 @@ __sync_checkpoint_can_skip(WT_SESSION_IMPL *session, WT_PAGE *page)
 static int
 __sync_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
 {
-	struct timespec end, start;
+	struct timespec end, start, cur_time;
 	WT_BTREE *btree;
 	WT_CONNECTION_IMPL *conn;
 	WT_DECL_RET;
 	WT_PAGE *page;
 	WT_REF *walk;
 	WT_TXN *txn;
+	uint64_t int_bytes_svpt, int_page_svpt, leaf_bytes_svpt, leaf_svpt;
```

Comment:
instead of leaf_svpt, we can have the name as leaf_pages_svft to increase readability.